### PR TITLE
store templates info in campaign

### PR DESCRIPTION
**Pull request by taran1898**
https://bitnudge.atlassian.net/l/cp/qU01dQEG

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -439,7 +441,7 @@
     t.bigint "contact_inbox_id"
     t.uuid "uuid", default: -> { "gen_random_uuid()" }, null: false
     t.string "identifier"
-    t.datetime "last_activity_at", default: -> { "CURRENT_TIMESTAMP" }, null: false
+    t.datetime "last_activity_at", default: -> { "now()" }, null: false
```
Comment (pararm9) [inline db/schema.rb:443]: why do we need to change this?

File: db/migrate/20231127191457_add_template_to_campaigns.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class AddTemplateToCampaigns < ActiveRecord::Migration[6.1]
+  def change
+    add_column :campaigns, :template, :jsonb, default: {}
+    add_column :campaigns, :template_parameters, :jsonb, default: {}
+    # add_column :campaigns, :selection_type, :enum selection_type {"label", "customer", "csv"}
```
Comment (pararm9) [inline db/migrate/20231127191457_add_template_to_campaigns.rb]: Lets remove this comment

File: app/views/api/v1/models/_campaign.json.jbuilder [hunk 3]
```diff
@@ -17,6 +17,7 @@ if resource.campaign_type == 'one_off'
   json.audience resource.audience
 end
 json.trigger_rules resource.trigger_rules
+json.template resource.template
```
Comment (pararm9) [inline app/views/api/v1/models/_campaign.json.jbuilder:20]: please add template_parameters & take care at other locations too

**Review by pararm9 — CHANGES_REQUESTED** [review]
Please attact doc as well

**Review by taran1898 — COMMENTED** [review]
remove gemfile changes

File: db/schema.rb (hunk 1 above)
Comment (taran1898) [reply db/schema.rb:443]: Set automatically by db migration command.

File: app/views/api/v1/models/_campaign.json.jbuilder (hunk 3 above)
Comment (taran1898) [reply app/views/api/v1/models/_campaign.json.jbuilder:20]: Not to be added as discussed.

**Review by pararm9 — APPROVED** [review]

**Review by pararm9 — APPROVED** [review]

**Pull request merged**